src/main/java/hudson/plugins/jira/JiraJobAction.java

```diff
@@ -129,6 +130,8 @@ public void onStarted(WorkflowRun workflowRun, TaskListener listener) {
                     setAction(parent, site);
                 } catch (IOException e) {
                     LOGGER.log(Level.WARNING, "Could not set JiraJobAction for <" + parent.getFullName() + ">", e);
+                } catch (RestClientException e) {
```

Comment:
Same here, why treat those differently?

Reply:
Here the `IOException` is being thrown from multiple places which I think is not specific to JiraRestClient.
For example: `job.save();` throws IOException. 
Another place from where it is thrown is: `String decodedJobName = URLDecoder.decode(job.getName(), "UTF-8");`

So I could handle those IOExceptions and rethrow as RestClientException.

Reply:
Why not `IOException e | RestClientException e` ?